OWNERS

```diff
@@ -3,4 +3,4 @@
 
 approvers:
   - adjackura
-  - hopkiw
+  - dorileo
```

Comment:
Since you are editing it please make it consistent with: https://github.com/GoogleCloudPlatform/guest-agent/blob/main/OWNERS